Simulation/Main.m

```diff
@@ -38,10 +40,10 @@
     break;
   endif
 endfor
-
 plot(Results.X, Results.Y)
 xlabel('X position (m)')
 ylabel('Y position (m)')
 set(legend('Trajectory'), "fontsize", 25)
 set(gca, "fontsize", 25)
-grid on
\ No newline at end of file
+grid on
+Save_Var_to_File(Results,results_file)
```

Comment:
This line can be removed, since the new simulation will be a function that outputs the "Results" we can save the results when we need to instead of every time we run a simulation.

Reply:
Line deleted